### PR TITLE
Output more feature properties

### PR DESCRIPTION
**Pull request by hiddewie**
Properties include:
- operator, traffic mode and radio for railway lines
- UIC ref for stations
- speed value for speed signals
- deactivated for speed and electricity signals
- voltage and frequency for electricity signals

### CODE REVIEW

File: import/openrailwaymap.lua [hunk 1]
```diff
@@ -604,6 +605,7 @@ function osm2pgsql.process_way(object)
       preserved_railway = tags['preserved:railway'],
       train_protection = railway_train_protection,
       train_protection_rank = railway_train_protection_rank,
+      operator = tags['operator'],
```
Comment (wolfy1339) [inline import/openrailwaymap.lua]: The operator tag can have a semicolon delimited list of operators. It isn't guaranteed to simply be a single operator.

It's a frequent thing in North America, where the primary operator/owner also leases access to other companies

**Comment by hiddewie** [comment]
electricity signals with frequency / voltage:

image

operator:

image

speed signal with speed:

image

UIC ref

image

File: import/openrailwaymap.lua (hunk 1 above)
Comment (hiddewie) [reply import/openrailwaymap.lua]: Nice, I will turn it into an array, so it can be displayed comma-separated in the UI!

Comment (hiddewie) [reply import/openrailwaymap.lua]: Done in d9b0b5eff39547652069ac5bbb861580cdbb8c2b

**Pull request merged**